src/scripts/macros/treat-wounds.ts

```diff
@@ -156,6 +156,7 @@ async function treat(
 
             if (riskySurgery) {
                 ChatMessagePF2e.create({
+                    flags: message.flags,
```

Comment:
Is this a message document or source? We'd want the source flags.

Reply:
It's the source now.